onnxruntime/test/contrib_ops/murmur_hash3_test.cc

```diff
@@ -49,14 +49,46 @@ TEST(MurmurHash3OpTest, ZeroSeedUIntResult2) {
   test.Run();
 }
 
-TEST(MurmurHash3OpTest, MoreData) {
+TEST(MurmurHash3OpTest, ZeroSeedUIntResult3) {
+  OpTester test("MurmurHash3", 1, onnxruntime::kMSDomain);
+  test.AddInput<int64_t>("X", {1}, {4LL});
+  test.AddAttribute<int64_t>("seed", 0LL);
+  test.AddOutput<uint32_t>("Y", {1}, {3491892518L});
+  test.Run();
+}
+
+TEST(MurmurHash3OpTest, ZeroSeedFloatResult) {
+  OpTester test("MurmurHash3", 1, onnxruntime::kMSDomain);
+  test.AddInput<float>("X", {1}, {3.});
+  test.AddAttribute<int64_t>("seed", 0LL);
+  test.AddOutput<uint32_t>("Y", {1}, {6814352L});
+  test.Run();
+}
+
+TEST(MurmurHash3OpTest, ZeroSeedDoubleResult) {
+  OpTester test("MurmurHash3", 1, onnxruntime::kMSDomain);
+  test.AddInput<double>("X", {1}, {3.});
+  test.AddAttribute<int64_t>("seed", 0LL);
+  test.AddOutput<uint32_t>("Y", {1}, {3554953595L});
```

Comment:
Where does it come from? Scikit-learn? Pytorch? Tensorflow?

Reply:
it's for supporting ML.NET(verified compatibility) and there is no source for test cases.based on offline chat